tests/ui/app_list.js

```diff
@@ -140,7 +140,14 @@ appListPages.forEach(function(appListPage) {
 
                 // Test expand toggle.
                 if (!appListPage.collection) {
-                    test.assertVisible('.expand-toggle');
+                    var toggleLink = '.app-list-filters-expand-toggle';
+                    test.assertVisible(toggleLink);
+                    casper.click(toggleLink);
+                    test.assertExists(toggleLink + '.active');
+                    test.assertExists('.app-list.expanded');
+                    casper.click(toggleLink);
+                    test.assertExists(toggleLink + ':not(.active)');
+                    test.assertExists('.app-list:not(.expanded)');
```

Comment:
Can you add a small bit of code that tests that the preview trays open up? I would put all the expand toggle and preview tray code in their own test ```casper.test.begin(appListPage.name + ' expand with previews tests')```